mentoring/tests/integration/test_mentoring.py

```diff
@@ -7,3 +10,45 @@ def test_display_submit_false_does_not_display_submit(self):
         mentoring = self.go_to_page('No Display Submit')
         with self.assertRaises(NoSuchElementException):
             mentoring.find_element_by_css_selector('.submit input.input-main')
+
+
+def _get_mentoring_theme_settings(theme):
+    return {
+        'package': 'mentoring',
+        'locations': ['public/themes/{}.css'.format(theme)]
+    }
+
+
+@ddt.ddt
+class MentoringThemeTest(MentoringAssessmentBaseTest):
+    def rgb_to_hex(self, rgb):
+        r, g, b = map(int, re.search(r'rgba?\((\d+),\s*(\d+),\s*(\d+)', rgb).groups())
+        return '#%02x%02x%02x' % (r, g, b)
+
+    def assert_checkmark_color(self, color):
```

Comment:
Can you please rename this test? "Checkmark" refers to these symbols: ✓, ✓, ☑ but this test is actually showing a wrong answer, which is a red ✗ - not a checkmark.

In fact the CSS class shared by checkmarks and X marks should probably be renamed to "status-icon" or something more generic.